viewing_party/main.py

```diff
@@ -0,0 +1,115 @@
+#WAVE 1 *******************************************************************************************************************************************************
+
+def create_movie(title, genre, rating):
+    while title and genre and rating:
+        return {"title":title, "genre":genre, "rating":rating}
+        
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for header in user_data["watchlist"]:
+        if header["title"] == title:
+            user_data["watchlist"].remove(header)
+            user_data["watched"].append(header)
+    return user_data
+
+
+#WAVE 2 *******************************************************************************************************************************************************
+
+def get_watched_avg_rating(user_data):
+    total = 0
+    if len(user_data["watched"]) == 0:
+        return 0
+    for movie in user_data["watched"]:
+        total += movie["rating"]
+    else:
+        return total/len(user_data["watched"])
+
+def get_most_watched_genre(user_data):
+    all_genres = {}
+    for movie in user_data["watched"]:
+        if movie["genre"] not in all_genres:
+            all_genres[movie["genre"]] = 1
+        else:
+            all_genres[movie["genre"]] += 1
+    if len(user_data["watched"]) == 0:
+        return None
+    most_watched = max(all_genres, key=all_genres.get)
+    return most_watched
+
+#WAVE 3 ******************************************************************************************************************************************************
+
+def get_unique_watched(user_data):
+    unseen_movies = []
+    movies_seen_by_friends = []
+    movies_seen_by_user = user_data["watched"]
+    for friends_watched in user_data["friends"]:
+        for friends_title in friends_watched["watched"]:
+            movies_seen_by_friends.append(friends_title)
+
+    for watched_title in movies_seen_by_user:
+        if watched_title in movies_seen_by_friends:
+            continue
+        unseen_movies.append(watched_title)
+            
+    return unseen_movies
+
+def get_friends_unique_watched(user_data):
+    unseen_movies = []
+    movies_seen_by_friends = []
+    movies_seen_by_user = user_data["watched"]
+    for friends_watched in user_data["friends"]:
+        for friends_title in friends_watched["watched"]:
+            movies_seen_by_friends.append(friends_title)
+
+    for watched_title in movies_seen_by_friends:
+        if watched_title not in unseen_movies:
+            if watched_title in movies_seen_by_user:
+                continue
+            unseen_movies.append(watched_title)
```

Comment:
let's put these two if statements on one line and flip it around to get rid of the `continue`:

```suggestion
        if watched_title not in unseen_movies and watched_title not in movies_seen_by_user:
             unseen_movies.append(watched_title)
```